Assets/scripts/Player/ControllerPlayer.cs

```diff
@@ -3,43 +3,76 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class ControllerPlayer
+public class ControllerPlayer:BaseController
 {
     
-    public ViewPlayer view ;
-    public ModelPlayer model;
-    public ServiceBullet serviceBullet;
+     ViewPlayer view ;
+     ModelPlayer model;
+     ServiceBullet serviceBullet;
+    InputComponent inputComponent;
+
     public ControllerPlayer(GameObject player,Transform spawnPoint)
     {
-
+      
         this.view = GameObject.Instantiate(player, spawnPoint.position,Quaternion.identity,null).GetComponent<ViewPlayer>();
         this.model =new ModelPlayer();
+        view.SetController(this);
         serviceBullet =ServiceBullet.Instance;
```

Comment:
no need for a local variable to store serviceBullet

Reply:
done